api/lanes.py

```diff
@@ -1071,8 +1074,12 @@ def __init__(self, library, collections=None):
             all Collections associated with `library` will be used.
         """
         self.library_id = library.id
+        self.collection_feed = False
         if collections:
             identifier = " / ".join(sorted([x.name for x in collections]))
+            if len(collections) == 1:
+                self.collection_feed = True
+                self.collection_name = collections[0].name
```

Comment:
This fixes the next links in the crawlable feeds. There's no route for a crawlable feed for multiple collections but not all the collections in a library, so this won't work for that case.